### DISCUSSION TITLE
How can I make 4 workspaces always available?

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [ldelossa/way-shell#31] [Q&A]
![image](https://github.com/ldelossa/way-shell/assets/98482469/5d389727-22dd-4dad-a35b-2e7ab934f9c7)
workspaces are only created when using sway shortcuts, I would like the 4 workspaces to always be available there

**Comment by ldelossa** [comment]
Way-Shell doesnt support "persistent" workspaces like WayBar right now. It will always reflect the current workspaces Sway is aware of. 

Im not too keen on persistent workspaces when no windows are in a workspace. What are the use cases? 

To create a new workspace you use the Workspace Switcher (super + w keybind)

**Reply by Heus-Sueh** [reply]
Personally, I prefer to click on the workspace instead of using the sway shortcut, a matter of habit not really efficiency

**Reply by ldelossa** [reply]
Sorry i don't think i understand, those buttons are clickable, and still switch you. What do you mean sway shortcuts?

**Reply by Heus-Sueh** [reply]
> Sorry i don't think i understand, those buttons are clickable, and still switch you. What do you mean sway shortcuts?

Persistent workspaces, as you said, as they have already been created they will always be shown in the workspace list even if they are empty
